optuna_dashboard/ts/components/AppDrawer.tsx

```diff
@@ -297,7 +297,7 @@ export const AppDrawer: FC<{
                 <ListItemIcon sx={styleListItemIcon}>
                   <RateReviewIcon />
                 </ListItemIcon>
-                <ListItemText primary="Note" sx={styleListItemText} />
+                <ListItemText primary="note" sx={styleListItemText} />
```

Comment:
I think you should not change this.


<img width="239" alt="Screenshot 2023-09-19 11 20 27" src="https://github.com/optuna/optuna-dashboard/assets/5564044/2deab6de-75f7-4afc-acbd-eb856069ec4a">

```suggestion
                <ListItemText primary="Note" sx={styleListItemText} />
```